.circleci/config.yml

```diff
@@ -13,43 +13,52 @@ jobs:
     steps:
       - checkout
       - restore_cache:
+          name: "Restore core dependencies cache"
           keys:
-            - dependencies-{{ checksum "yarn.lock" }}
+            - core-dependencies-{{ checksum "yarn.lock" }}
       - run:
           name: "Install dependencies"
           command: |
             yarn install
       - run:
-          name: "Bootstrapping"
+          name: "Dirty check"
           command: |
-            yarn bootstrap --core --docs --reactnative --reactnativeapp
+            yarn repo-dirty-check
       - run:
-          name: "Dirty checking"
+          name: "Bootstrap"
           command: |
-            yarn repo-dirty-check
+            yarn bootstrap --core
       - save_cache:
-          key: dependencies-{{ checksum "yarn.lock" }}
+          name: "Cache core dependencies"
+          key: core-dependencies-{{ checksum "yarn.lock" }}
           paths:
             - node_modules
-            - docs/node_modules
-            - examples/react-native-vanilla/node_modules
-            - examples/crna-kitchen-sink/node_modules
+            - example/cra-kitchen-sink/node_modules
+            - example/vue-kitchen-sink/node_modules
+      - save_cache:
+          name: "Cache core dist"
+          key: core-dist-{{ .Revision }}
+          paths:
+            - addons
+            - app
+            - lib
```

Comment:
This caches `app`, `addons`, and `lib` directories using current commit hash as a part of the cache key.

`bootstrap --core` transpiles all the packages in those directories (`src` to `dist`).

Ideally, it should be
```
paths:
  - addons/*/dist
  - app/*/dist
  - lib/*/dist
```

But unfortunately, wildcards are not supported in cached directories paths